pyvo/dal/tap.py

```diff
@@ -922,10 +922,9 @@ def wait(self, phases=None, timeout=600.):
             if cur_phase in phases:
                 break
 
-            # fallback for uws 1.0
-            if Version(self._job.version) < Version("1.1"):
```

Comment:
Could you provide a failing example that argues for removing this conditional as a fix?